treedoc/traversal.py

```diff
@@ -378,7 +382,7 @@ def unique_first(gen1, gen2):
 
 
 def is_inspectable(obj) -> bool:
-    """An object is inspectable if it returns True for any of the inspect.is.. functions."""
+    """An object is inspectable if it returns True for any of the inspect.is* functions."""
```

Comment:
@tommyod : Do you agree with this notation?

Reply:
Yes, better than mine.